components/Wizard/ReviewStep.js

```diff
@@ -183,7 +186,7 @@ class ReviewStep extends React.PureComponent {
             bodyContent={
               <FormattedMessage
                 defaultMessage="
-                      Image Builder can upload images you create to a Blob container in {azure}. When the image build is complete 
+                      Image Builder can upload images you create to a Blob container in {azure}. When the image build is complete
```

Comment:
Can you update the aws test to not include the extra space too. If you have any questions about the test let me know!